src/components/StatTile.vue

```diff
@@ -198,6 +241,25 @@ export default class StatTile extends Vue {
     return null;
   }
 
+  // Returns the inverse of a rank, so the # lowest in a category
+  // E.g rank #100 Highest/100 total in GHG intensity is #1 Lowest
+  // Only returns a rank if there are 80 or more buildings per rank
+  // (50 highest and 30 lowest)
+  get statRankInvertedByProperty(): number | null {
+    const buildingStatsByPropertyType = 
```

Comment:
Generally you want to have imports at the top of a file, and I think you can import this JSON like we do in other files, you shouldn't need to do a `require`.